src/test/scala/org/broadinstitute/hail/stats/symEigDSuite.scala

```diff
@@ -0,0 +1,85 @@
+package org.broadinstitute.hail.stats
+
+import breeze.linalg.{DenseMatrix, eigSym, svd}
+import breeze.stats.distributions.{RandBasis, ThreadLocalRandomGenerator}
+import org.apache.commons.math3.random.MersenneTwister
+import org.broadinstitute.hail.SparkSuite
+import org.testng.annotations.Test
+import org.broadinstitute.hail.stats._
+import org.broadinstitute.hail.utils._
+
+class symEigDSuite extends SparkSuite {
+  @Test def eigSymTest() = {
+    val seed = 0
+    implicit val rand: RandBasis = new RandBasis(new ThreadLocalRandomGenerator(new MersenneTwister(seed)))
+
+    val n = 5
+    val m = 10
+    val W = DenseMatrix.fill[Double](n, m)(rand.gaussian.draw())
+    val K = W * W.t
+
+    val svdW = svd(W)
+    val svdK = svd(K)
+    val eigSymK = eigSym(K)
+    val eigSymDK = eigSymD(K)
+    val eigSymRK = eigSymR(K)
```

Comment:
Testing different implementations are great.  You might also test a small, concrete input.  If everything is systematically wrong you won't catch it.